.github/workflows/label-conflicting-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: "Label conflicting PRs"
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+    
+permissions:
+  pull-requests: write
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label conflicting PRs
+        uses: eps1lon/actions-label-merge-conflict@releases/2.x
```

Comment:
did we want to use a fixed commit?

Reply:
That's what the docs of the action recommend, want to unpin it?

Reply:
I think may as well for a more 'random' action given the permissions.

the others we use here are either ones in `actions`, `release-drafter` or jenkinsci

Reply:
I think @timja is suggesting to use a SHA here? Could you clarify?